web/groups/hlt.json

```diff
@@ -18,6 +18,12 @@
   "CAHitNtupletCUDA|hltPixelTracksGPU": "Pixels|GPU",
   "CAHitNtupletCUDA|hltPixelTracksHitQuadruplets": "Pixels|GPU",
   "CAHitNtupletCUDA|hltPixelTracksSoA*": "Pixels",
+  "CAHitNtupletCUDAPhase1|hltPixelTracksCPU": "Pixels",
+  "CAHitNtupletCUDAPhase1|hltPixelTracksCPUOnly": "Pixels",
+  "CAHitNtupletCUDAPhase1|hltPixelTracksCUDA": "Pixels|GPU",
+  "CAHitNtupletCUDAPhase1|hltPixelTracksGPU": "Pixels|GPU",
+  "CAHitNtupletCUDAPhase1|hltPixelTracksHitQuadruplets": "Pixels|GPU",
```

Comment:
do we have this module ?
I do not see it in the latest HLT menu (in CMSSW)

Reply:
That is interesting! It does show up in the `resources.json` file of my recent CMSSW_13_0_0_pre3 measurement. E.g.:
```
    {
      "events": 7725,
      "label": "hltPixelTracksGPU",
      "mem_alloc": 19748,
      "mem_free": 16541,
      "time_real": 2659.943584,
      "time_thread": 2218.270178,
      "type": "CAHitNtupletCUDAPhase1"
    },
```

Reply:
I meant specifically `hltPixelTracksHitQuadruplets`.

Reply:
I brought it up because the name is generic enough that the module could be something running on CPU or GPU... and when I tried to check, I could not find it at all.

Reply:
Done.

Reply:
Thanks!